sci-visualization/mricrogl/mricrogl-1.0.20170714.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit gnome2-utils
+
+DESCRIPTION="A simple medical imaging visualization tool"
+HOMEPAGE="https://github.com/neurolabusc/MRIcroGL"
+SRC_URI="https://github.com/neurolabusc/MRIcroGL/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND=""
+DEPEND="dev-lang/fpc
```

Comment:
doesn't fpc also provide the runtime? AFAIK fpc-based programs aren't fully statically linked